server/index.js

```diff
@@ -1,18 +1,118 @@
 const express = require('express');
 const path = require('path');
-const { db, sync } = require('./db/db');
+const cookieParser = require('cookie-parser');
+const chalk = require('chalk');
+const dotenv = require('dotenv');
+const { sync } = require('./db/db');
+const routes = require('./routes');
+const { Session, User, Cart } = require('./db/models');
+const { noDirectAccess, adminApiSecurityCheck, accessDeniedResponse } = require('./utils')
+
 
 const app = express();
 const PORT = process.env.PORT || 3000;
 
-app.use(express.static(path.join(__dirname, '../dist')))
+app.use(express.json());
+app.use('/client/assets', express.static(path.join(__dirname, '../client/assets')));
+app.use(express.static(path.join(__dirname, '../dist')));
+app.use(cookieParser());
+
+dotenv.config();
+
+app.use(async (req, res, next) => {
```

Comment:
index of server should be simpler, can move this session middleware to solo file.